hybridse/src/udf/default_udf_library.cc

```diff
@@ -1612,6 +1612,21 @@ void DefaultUdfLibrary::InitTimeAndDateUdf() {
             @since 0.1.0
         )");
 
+    RegisterExternal("dayofyear")
+        .args<int64_t>(static_cast<int32_t (*)(int64_t)>(v1::dayofyear))
+        .args<Timestamp>(static_cast<int32_t (*)(Timestamp*)>(v1::dayofyear))
+        .args<Date>(static_cast<int32_t (*)(Date*)>(v1::dayofyear))
+        .doc(R"(
+            @brief Return the day of year for a timestamp or date.
+
+            Example:
+            @code{.sql}
+                select dayofyear(timestamp(1590115420000));
```

Comment:
Good Job!
Please add demo SQL for dayofyear(int64 type value) and dayofyear(date type value) as well.

Reply:
thanks, fixed!